### PR TITLE
Cerestation Tweaks VI: Shuttle Edition

### PR DESCRIPTION
**Pull request by MMMiracles**
- Added a new escape shuttle design for Cerestation. Holds over 80 people with several fully-stocked sub-departments for people to kill themselves with at the end of the round.
- Added a new white shuttle design for Cerestation. Meant mostly as a cargo vessel for transporting materials during the station's construction. Comes with a spawner that has a 75% chance to spawn ripley wreckage or 25% chance to spawn a fully-equipped ripley.
- Fixed several cargo turfs that had lava baseturfs.
- Added the trophy case to the Library.
- All SMES rooms are now considered maintenance areas with some slight reworking to prevent redundant extra areas/APCs. Maint-like areas that aren't rad-protected (i.e the external airlock rooms) now have a special maintenance door with a stripe with a similar color design to external airlocks. Hopefully helps cut down on confusion with rad-immune areas.

**Escape Shuttle**
shutt
- Can easily hold over 80 people with room to run around in. 
- Includes fully-stocked medbay with 4 sleepers, blood and IV drip, one of each medkit, a full surgery set, and a functional cryotube. If you're gonna be treating 60-70+ people, you might as well be well-stocked for it.
- Cargo hold with mech rechargers for science/mining to park their stompy mechs and charge up.
- Expanded brig section with separate seating areas for officers and prisoners. Includes 2 chargers, spare zipties, pepperspray refiller, and a box of teargas grenades for when the shuttle gets too rowdy.
- Engine room for engineers to pretend to be monitoring the shuttle and grab some supplies for when the inevitable suicide-bomber blows a chunk out of the shuttle. 
- Bathroom section with a borg recharger to do last-minute shady shit, like murdering that target you forgot about.
- Complimentary donkpockets and water.

**Whiteship**
whites
- Parks at an external airlock on the bottom section of the Engineering asteroid.
- Comes with some basic materials and tools.
- 2 mech chargers.
- Spawner with a 75% to spawn ripley debris and a 25% to spawn a fully-equipped mining ripley.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
> Remember to add lifeboats onto this HUGE shuttle for women, children & Nekos first.

> Play em out with a musical instrument cabinet when this inevitably sinks in hyperspace.

> Watch out for that space Iceberg

File: tgstation.dme [hunk 1]
```diff
@@ -673,10 +673,10 @@
...
 #include "code\game\objects\effects\decals\misc.dm"
 #include "code\game\objects\effects\decals\remains.dm"
-#include "code\game\objects\effects\decals\cleanable\aliens.dm"
-#include "code\game\objects\effects\decals\cleanable\humans.dm"
-#include "code\game\objects\effects\decals\cleanable\misc.dm"
-#include "code\game\objects\effects\decals\cleanable\robots.dm"
+#include "code\game\objects\effects\decals\Cleanable\aliens.dm"
+#include "code\game\objects\effects\decals\Cleanable\humans.dm"
+#include "code\game\objects\effects\decals\Cleanable\misc.dm"
+#include "code\game\objects\effects\decals\Cleanable\robots.dm"
```
Comment (coiax) [inline tgstation.dme]: I think you made this from an old branch.

**Comment by Armhulen** [comment]
White ship more like WHITE SHIT REDO IT

**Comment by francinum** [comment]
Unless this is limited strictly to Cerestation, most stations are going to need another redesign to the escape area to make this work.

**Comment by francinum** [comment]
Also, picture of the new airlock.

**Comment by FantasticFwoosh** [comment]
Dunno francium, the gung-ho crash into the side of the station approach never held back the asteroid ship.

**Comment by francinum** [comment]
True, but unless you want pubbystation to lose arrivals when the ship docks something's gonna have to change.

**Comment by MMMiracles** [comment]
As Deltastation's shuttle would say:

_"Go big or go home"_

**Comment by francinum** [comment]
>starts booting a copy of the branch
Let me show you something about that...

**Comment by FantasticFwoosh** [comment]
Pretty please can we call the ship the Titanic.

**Comment by francinum** [comment]
Yeah, because matching up docking ports you're gonna crash right the fuck into most stations.

**Comment by francinum** [comment]
Case In Point.

**Comment by MMMiracles** [comment]
I guess  you probably shouldn't drop 4.5k credits on a shuttle that you know won't fit the current station's dock.

**Comment by francinum** [comment]
for fucks sake atleast put your docking port in the correct place.

**Comment by coiax** [comment]
Yeah, traditionally the docking port goes in the second from top airlock.

**Comment by MMMiracles** [comment]
Docking port fixed.

**Comment by francinum** [comment]
Now it crashes into everything slightly less.

**Comment by MMMiracles** [comment]
Works just fine on stations that don't have their docking port indented into the station, like Box.

**Comment by francinum** [comment]
Why did you use box as an example...

**Comment by MMMiracles** [comment]
They didn't need those freezers anyway.

**Comment by francinum** [comment]
The point is, lock it to cerestation, or the stations need to be modified to accept it, the Asteroid gets a pass because crushing everything is it's gimmick.

File: tgstation.dme [hunk 2]
```diff
@@ -674,10 +674,10 @@
 #include "code\game\objects\effects\decals\decal.dm"
 #include "code\game\objects\effects\decals\misc.dm"
 #include "code\game\objects\effects\decals\remains.dm"
-#include "code\game\objects\effects\decals\cleanable\aliens.dm"
-#include "code\game\objects\effects\decals\cleanable\humans.dm"
-#include "code\game\objects\effects\decals\cleanable\misc.dm"
-#include "code\game\objects\effects\decals\cleanable\robots.dm"
+#include "code\game\objects\effects\decals\Cleanable\aliens.dm"
```
Comment (lzimann) [inline tgstation.dme]: Fix the folders, please

**Comment by Tacolizard** [comment]
we need a Universal Docking Standard™ so that we don't have to deal with this stuff anymore.

Also the white ship is really barren. It needs some medkits or something. A ship meant for interstellar travel should have at least basic amenities.

**Comment by MMMiracles** [comment]
Cerestation's white shuttle is meant to be more of a cargo shuttle that was used for building the station.

I'd rather get this PR merged with both shuttles so the fixes/main shuttle gets in and then I can spend time later making a more interesting white shuttle if people go 'meh' towards it.

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -161,6 +161,13 @@
 	description = "A small, but feature complete shuttle. It boasts a card table to keep crew members occupied on the long flight home."
 	credit_cost = 1000
 
+/datum/map_template/shuttle/emergency/cere
+	suffix = "cere"
+	name = "Cere Station Emergency Shuttle"
+	description = "The large, beefed-up version of the box-standard shuttle. Includes an expanded brig, fully stocked medbay, enhanced cargo storage with mech chargers, \
+		an engine room stocked with various supplies, and a crew capacity of 80+ to top it all off. Live large, live Cere."
+	can_be_bought = FALSE
```
Comment (lzimann) [inline code/datums/shuttles.dm]: You can just remove the price(it is set to `INFINITY` at base) and it will be fine. No need to use this var.

Comment (MMMiracles) [inline code/datums/shuttles.dm]: Does it really matter? If its limited to only Cerestation, so why even bother showing it in the buy menu?

Comment (lzimann) [inline code/datums/shuttles.dm]: If the price is infinity, it won't show up.

File: code/datums/shuttles.dm [hunk 4]
```diff
@@ -161,6 +161,13 @@
 	description = "A small, but feature complete shuttle. It boasts a card table to keep crew members occupied on the long flight home."
 	credit_cost = 1000
 
+/datum/map_template/shuttle/emergency/cere
+	suffix = "cere"
+	name = "Cere Station Emergency Shuttle"
+	description = "The large, beefed-up version of the box-standard shuttle. Includes an expanded brig, fully stocked medbay, enhanced cargo storage with mech chargers, \
+		an engine room stocked with various supplies, and a crew capacity of 80+ to top it all off. Live large, live Cere."
+	credit_cost = INFINITY
```
Comment (lzimann) [inline code/datums/shuttles.dm]: One last thing and I'll merge it! This is redundant, it's already set in the `map_template/shuttle` base ([here](https://github.com/tgstation/tgstation/blob/master/code/datums/shuttles.dm#L12)) !

**Pull request merged**